src/ladds/particle/Particle.cpp

```diff
@@ -18,8 +17,8 @@ double Particle::calculateBcInv(double bstar, double mass, double radius, double
     // or via bstar
     // @note see https://en.wikipedia.org/wiki/BSTAR
     // B* == p_0 * c_D * A / (2 m) == bc_inv * p_0 / 2
-    constexpr double p0Inv = 1. / (2.461 * 1e-5 * Physics::R_EARTH);  // 1/(kg/(m^2 * R_EARTH)) == R_EARTH * m^2/kg
-    return bstar * 2. * p0Inv;                                        // m^2/kg
+    // Thus we factor out p0 (=0.1570), convert to m (original bstar is in Earth radii) and the factor 2
+    return 2.0 * bstar * Physics::R_EARTH / 0.1570;
```

Comment:
```suggestion
    constexpr factor = 2.0 * Physics::R_EARTH / 0.1570;
    return factor * bstar;
```
Move constant computations to compile time